src/enums/index.ts

```diff
@@ -0,0 +1,125 @@
+import { makeClass } from './../utils/make-class'
```

Comment:
This file was just renamed + updated import/export syntax

Reply:
SGTM. I'm not going to comment on styles in this file then.